week-1-practise/level-1/1-syntax-error-1.js

```diff
@@ -1,14 +1,13 @@
 // There is a syntax error in this code. Fix it.
 
-function addNumbers(a b c) {
-    return a + b + c;
+function addNumbers(a, b, c) {
+  return a + b + c;
 }
 
 var result = addNumbers(1, 3, 4);
 console.log(result); // Expected result 8 - Run the program and make sure it works
 
-
 // Answer these questions:
 // 1. How many parameters does the function take?
 // 2. What is the function name?
-// 3. Where is the function called? with what parameters?
\ No newline at end of file
+// 3. Where is the function called? with what parameters?
```

Comment:
// There is a syntax error in this code. Fix it.

function addNumbers(a, b, c) {
  return a + b + c;
}

var result = addNumbers(1, 3, 4);
console.log(result); // Expected result 8 - Run the program and make sure it works

// Answer these questions:
// 1. How many parameters does the function take?
// 2. What is the function name?
// 3. Where is the function called? with what parameters?
/*Respose */

/* -The function take three parameters
-The function name is: addNumbers
- The is called in line 7 and (1,2,3) as parameters */

Reply:
No problems with it for the most part, but you might like to check the parameters in your answer to Q3